### DISCUSSION TITLE
How to see context of translation

### DISCUSSION THREAD

**Discussion by Znerox** [babybuddy/babybuddy#882] [Q&A] [answered]
I'm cleaning up a a translation, and find it hard to see where each phrase is used. I see a reference on poeditor, but that only show the source file. Is there any tricks to easily find the translated phrase in Baby Buddy?

**Comment by cdubz** [comment, marked as answer]
Hmmm, yeah the info in POEditor is actually pretty good. But it's not in a useful format. Really the only way to do what you want to do would be to search the code base for the English string.
